test/cppsuite/test_harness/test.cpp

```diff
@@ -162,7 +161,8 @@ test::run()
 
     /* Validation stage. */
     if (_workload_tracking->enabled())
-        this->validate(_workload_tracking->get_operation_table_name(),
+        this->validate(_config->get_subconfig(WORKLOAD_TRACKING)->get_bool("is_custom"),
```

Comment:
Please use the constant for `is_custom`

Reply:
Fixed in ee0d40bb4aec6f0ebee5f97b330156f4a1449f12